README.md

```diff
@@ -1,103 +1,89 @@
-| Start dev mode in a container                        | Starts dev mode with the server in a container. The `liberty-maven-plugin` must be version `3.3-M1` or higher. The `liberty-gradle-plugin` must be version `3.1-M1` or higher.
-| Stop dev mode                         | Stops dev mode.                                                                                                                                                                                                                                                                                                              |
-| Run dev mode tests                    | Runs the unit tests and integration tests that are configured for your project. This command requires dev mode to be already started.                                                                                                                                                                                        |
-| View integration test report | Views the integration test report file if it exists at `/target/site/failsafe-report.html`.                                                                                                                                                                                                                                                                                      |
-| View unit test report        | Views the unit test report file if it exists at `/target/site/surefire-report.html`.                                                                                                                                                                                                                                                                                             |
-
-**Note:** Gradle projects only have a single `View test report` command. This will open the report if a file can be
-found that matches the configured `tests.reports.html.destination` in the `build.gradle` file, otherwise it will open
-the default `index.html` file.
+![LSP4Jakarta completion](docs/images/LSP4Jakarta-rest-completion.png)
 
-## Contributing
+Code completion when editing microprofile-config.properties files helps you easily set properties for MicroProfile APIs.
 
-Contributions to the Liberty Tools extension are welcome!
+![LSP4MP completion](docs/images/LSP4MP-mp-properties-completion.png)
 
-Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pull requests.
+Editing assistance for Liberty configuration files is provided through the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server). For more information, see the [project documentation in GitHub](https://github.com/OpenLiberty/liberty-language-server#liberty-config-language-server).
```

Comment:
wonder if we can reduce some redundancy here.

Editing assistance for configuration and application files is provided through the following language server projects, which this project consumes. For more information, see the documentation for these projects.

- Liberty configuration files: [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server)
- Jakarta EE APIs in Java files: [Eclipse LSP4Jakarta](https://github.com/eclipse/lsp4jakarta), the Language Server for Jakarta EE
- MicroProfile APIs in microprofile-config.properties and Java files: [Eclipse LSP4MP](https://github.com/eclipse/lsp4mp), the Language Server for MicroProfile

Reply:
Updated